### ISSUE TITLE
Trend line does not appear on H4 charts

### ISSUE THREAD

**Issue by robustify** [louisnw01/lightweight-charts-python#66]
Hello again! I'm trying to draw trend lines on a chart with the `trend_line` method, but the line does not appear on four-hour charts. It does appear on one-hour and daily charts, however.

Here are some small CSV files that can be used to show a minimal example of this behavior:

[daily_bar_data.csv](https://github.com/louisnw01/lightweight-charts-python/files/12328371/daily_bar_data.csv)
[four_hour_bar_data.csv](https://github.com/louisnw01/lightweight-charts-python/files/12328372/four_hour_bar_data.csv)
[one_hour_bar_data.csv](https://github.com/louisnw01/lightweight-charts-python/files/12328373/one_hour_bar_data.csv)

and here is the minimum example code to load the above CSV files and draw a trend line between the 6th and 16th bar open prices:

```
chart_data = pd.read_csv('one_hour_bar_data.csv', index_col=0)
# chart_data = pd.read_csv('four_hour_bar_data.csv', index_col=0)
# chart_data = pd.read_csv('daily_bar_data.csv', index_col=0)
chart = Chart()
chart.precision(5)
chart.set(chart_data)
chart.trend_line(
    start_time=chart_data.index.values[5],
    start_value=chart_data.iloc[5].open,
    end_time=chart_data.index.values[15],
    end_value=chart_data.iloc[15].open,
    color='#ffffff', width=2
)
chart.show(block=True)
```

H1 bars work:
![h1_works](https://github.com/louisnw01/lightweight-charts-python/assets/8763875/02af3ebf-31d7-4446-b8b4-07e190f8ae05)

Daily bars work:
![daily_works](https://github.com/louisnw01/lightweight-charts-python/assets/8763875/baacb749-9807-4625-9ff9-a86caa2102dc)

H4 bars don't draw the line:
![h4_no_line](https://github.com/louisnw01/lightweight-charts-python/assets/8763875/3431d08a-1c77-45c3-a2c9-a746804fdcff)

**Comment by louisnw01** [comment]
Hey @robustify,

Thanks for raising this; this appears to be a rounding issue and an edge case with non-standard timeframes. Will be fixed for the next update.

Louis

**Comment by louisnw01** [comment]
Fixed in the latest version (1.0.16).

Louis
